qa/rpc-tests/wallet_accounts.py

```diff
@@ -173,11 +173,10 @@ def run_test(self):
         txid = wait_and_assert_operationid_status(self.nodes[0], opid)
 
         # The wallet should detect the new note as belonging to the UA.
-        # TODO: Uncomment once z_viewtransaction shows Orchard details (#5186).
-        #tx_details = self.nodes[0].z_viewtransaction(txid)
-        #assert_equal(len(tx_details['outputs']), 1)
-        #assert_equal(tx_details['outputs'][0]['type'], 'orchard')
-        #assert_equal(tx_details['outputs'][0]['address'], ua0)
+        tx_details = self.nodes[0].z_viewtransaction(txid)
```

Comment:
There are more test lines to uncomment below.

Reply:
If CI passes, I'll submit that as a separate PR.